content/applications/services/helpdesk/advanced/close_tickets.rst

```diff
@@ -1,48 +1,98 @@
-To designate which stage the ticket migrates to once it is closed, navigate to the ticket pipeline
-by going to :menuselection:`Helpdesk --> Overview` and clicking :guilabel:`Tickets` on the team's
-card.
+   .. image:: close_tickets/closing_folded_setting.png
+      :align: center
+      :alt: Stage settings page
 
-There are two options: create a new Kanban stage or work with an existing one. For both scenarios,
-click the :guilabel:`Settings (gear)` icon next to the stage name, select :guilabel:`Edit Stage`,
-and enable :guilabel:`Closing Stage`. After checking the field box, click :guilabel:`Save` to
-finish.
 
-.. note::
-   If a closing stage is not specified, by default, the ticket is moved to the last stage in the
-   kanban. If more than one stage is set as a closing stage, the ticket is placed in the first
-   closing stage column.
+Automatic closing
+=================
 
-With the ticket closing settings now complete, customers can now view the option to
-:guilabel:`Close this ticket` when they log into their portal.
+Tickets that are inactive for a set period of time, can be automatically closed. At that point,
+they will be moved to a folded stage.
 
-.. image:: close_tickets/customer-view-close-ticket.png
-   :alt: Customer view of ticket closing in Odoo Helpdesk.
+Go to the team's settings page by going to :menuselection:`Helpdesk --> Configuration --> Teams`.
+Under the :guilabel:`Self-Service` section, enable :guilabel:`Automatic Closing`.
+
+.. image:: close_tickets/closing_automatic_setting.png
+   :align: center
+   :alt: Automatic ticket closing setting options on team page
+
+If one of the team's stages is set to be folded in the kanban view, it will be the default selection
+in the :guilabel:`Move to Stage` field. If the team has more than one folded stage, the stage that
+occurs first in the pipeline will be the default. If no stage is set to be folded, the default
+selection will be the last stage in the pipeline.
+
+Adjust the :guilabel:`After days of inactivity` field to the appropriate number of days.
+
+.. important::
+   The :guilabel:`After days of inactivity` field does not take the working calendar into account
+   when tracking time.
+
+If only certain stages should be used to track days of inactivity, they can be added in the
+:guilabel:`In Stages` field.
+
+.. example::
+   A pipeline has the following stages:
+
+   - New
+   - In Progress
+   - Customer Feedback
+   - Closed
+
+   Tickets may linger in the *Customer Feedback* stage, because once an issue is solved, customers do
+   not respond right away. At that point, they can be closed automatically. However, tickets in the
+   *New* and *In Progress* stages may remain inactive due to assignment or workload issues. Closing
+   them automatically may result in issues going unsolved.
 
-Get reports on tickets closed by customers
-==========================================
+   Therefore, the :guilabel:`Automatic Closing` settings would be configured as below-
```

Comment:
```suggestion
   Therefore, the :guilabel:`Automatic Closing` settings would be configured as below:
```

Reply:
Thank you for catching this! It was formatting oddly because of the guilabel markup (adding a colon at the end of the line was causing the entire line to render oddly). I had to add a backslash to get it to render properly. Fixed 👍